lib/script/common.js

```diff
@@ -414,6 +414,12 @@ exports.hashType = {
 
   SINGLE: 3,
 
+  /*
+   * Sign output at the n-i'th index (zero sequences).
```

Comment:
I don't love this docstring, thinking of a better way to explain it

Reply:
Updated the docstring with something better. Its a bit longer but should be good enough for people to understand.